daemon/logger/splunk/splunk.go

```diff
@@ -0,0 +1,256 @@
+// +build linux
```

Comment:
@LK4D4 the Splunk instance can be running on any platform. The driver is just making simple HTTP request, so I don't see why it would not work on Windows.